Sources/StreamChatSwiftUI/ChatChannel/ChatChannelViewModel.swift

```diff
@@ -89,6 +89,15 @@ open class ChatChannelViewModel: ObservableObject, MessagesDataSource {
         }
     }
 
+    public private(set) var bouncedMessage: ChatMessage?
```

Comment:
Shouldn't this be `@Published`?

Reply:
At the moment I don't need it to be, but could be useful for customers maybe 🤔

Reply:
Done